cloud/cluster/memory/node.go

```diff
@@ -0,0 +1,48 @@
+package memory
+
+import (
+	"fmt"
+	"github.com/scootdev/scoot/cloud/cluster"
+)
+
+type idNode struct {
+	id cluster.NodeId
+}
+
+func (n *idNode) Id() cluster.NodeId {
+	return n.id
+}
+
+func (n *idNode) String() string {
+	return string(n.id)
+}
+
+func NewIdNode(id string) cluster.Node {
+	return &idNode{cluster.NodeId(id)}
+}
+
+var _ cluster.Node = (*idNode)(nil)
+
+func NewIdNodes(num int) []cluster.Node {
```

Comment:
Can you tell me more about that?

Calling NewIdNodes doesn't add it to our pol the first time. If we want to add, we can call NewIdNodes, and then wrap them in NodeUpdates and send them in to the channel.

I agree about "fake" versus "valid", which is why I haven't called any of these fake. They're in-memory implementations.